test/e2e/load.go

```diff
@@ -74,10 +74,6 @@ var _ = Describe("Load capacity", func() {
 
 	// TODO add flag that allows to skip cleanup on failure
 	AfterEach(func() {
```

Comment:
I think this might have issues, since the Framework's AfterEach will have already deleted everything?